src/util/prom_types.py

```diff
@@ -18,6 +18,7 @@
 PROM_THIRDPARTY_METRICS = getConfig('PROM_THIRDPARTY_METRICS', "").split(',')
 
 metric_prefix = "kepler_"
+custom_metrics = "node_cpu_seconds_total"
```

Comment:
If this is fixed, it should be defined in feature group.
If it is custom definition, it should be able to set dynamically by design of third-party metric introduced by this PR https://github.com/sustainable-computing-io/kepler-model-server/pull/185.

Reply:
I agree @sunya-ch , there is no need of `custom_metrics` since third-party metric helps achieve the same.